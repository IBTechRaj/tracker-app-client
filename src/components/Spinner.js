import React from 'react';
import spinner from './spinner.gif';

function Spinner() {
  return (
    <div>
      <img
        src={spinner}
        style={{
          width: '350px', margin: 'auto', height: '65vh', display: 'block',
        }}
        alt="Loading..."
      />
    </div>
  );
}

export default Spinner;
