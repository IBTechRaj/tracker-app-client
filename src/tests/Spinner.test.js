import React from 'react';
import renderer from 'react-test-renderer';
// import ReactTestUtils from 'react-dom/test-utils'; // ES6
import Spinner from '../components/Spinner';

test('renders learn react link', () => {
  const component = renderer.create(<Spinner />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
