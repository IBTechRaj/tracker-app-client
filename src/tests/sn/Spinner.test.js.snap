// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <img
    alt="Loading..."
    src="spinner.gif"
    style={
      Object {
        "display": "block",
        "margin": "auto",
        "width": "200px",
      }
    }
  />
</div>
`;
